openshift-start-configured-cluster

```diff
@@ -12,6 +12,9 @@ DIR=`readlink -f $DIR`
 source $DIR/config
 
 OC_CONFIG=$DIR/openshift-basedir
+if [ -d "$OC_CONFIG" ]; then
+  ./cleanup
+fi
```

Comment:
We sure we want to do this implicitly ?

Reply:
The startup script as it is now, fails for me if I try to run it twice (`./openshift-start-configured-cluster && oc cluster down && ./openshift-start-configured-cluster`. 

But after your comment I have some second thoughts about this. Let me check if I can get it working with the scenario described above. Will let you know in a couple of minutes.

Reply:
So I ran a couple of tests and I would vote for doing the cleanup here. 

Running `./openshift-start-configured-cluster && oc cluster down && ./openshift-start-configured-cluster` would require additional checking in almost every step (certificate generation, instantiating Keycloak etc). IMO, the best way is to just clean it up before booting up the script.

Reply:
ok